compose-table/src/main/java/org/dhis2/composetable/ui/TextInputUi.kt

```diff
@@ -61,7 +61,8 @@ fun TextInput(
     tableColors: TableColors? = null,
     onTextChanged: (TextInputModel) -> Unit,
     onSave: () -> Unit,
-    onNextSelected: () -> Unit
+    onNextSelected: () -> Unit,
+    focusRequester: () -> FocusRequester
```

Comment:
Why are you using focusRequester as a function parameter and not using the FocusRequester object directly?

Reply:
Your're absolutely right! changes are made